src/ioc-react/mapDependenciesToProps.ts

```diff
@@ -0,0 +1,17 @@
+import { Container } from '../ioc';
+
+/**
+ * @ignore
+ */
+export function mapDependenciesToProps<T>(container: Container, inject: T): { [propName in keyof T]: any } {
+  const dependencies = {} as any;
```

Comment:
minor, but since this is the same as the return signature, could we do 

`interface Props = { [propName in keyof T]: any }`

and then use that both in the return signature and here in line 7